controllers/defaults/resources.go

```diff
@@ -122,12 +122,12 @@ var (
 	BalancedDaemonResources = map[string]corev1.ResourceRequirements{
 		"mgr": {
 			Requests: corev1.ResourceList{
-				corev1.ResourceCPU:    resource.MustParse("1"),
-				corev1.ResourceMemory: resource.MustParse("1.5Gi"),
+				corev1.ResourceCPU:    resource.MustParse("1.5"),
+				corev1.ResourceMemory: resource.MustParse("3Gi"),
 			},
 			Limits: corev1.ResourceList{
-				corev1.ResourceCPU:    resource.MustParse("1"),
-				corev1.ResourceMemory: resource.MustParse("1.5Gi"),
+				corev1.ResourceCPU:    resource.MustParse("1.5"),
```

Comment:
@travisn ^^

Reply:
/hold until this query is resolved

Reply:
> CPU is raised from 1 to 1.5 between 4.14 and 4.15 is it expected?

I believe that was proposed with the new profiles. But now that we are reevaluating the resources, a goal I would propose is to make the balanced profile requests as close to 4.14 as possible so we don't have a regression in resource requests. Thus I would propose:
1. Change the cpu requests for the balanced profile back to `1`
2. Change the cpu requests for the basic profile back to `0.5`
3. Set **double the memory limits** compared to the memory requests
4. Set **double the cpu limits** compared to the cpu requests
5. Memory requests for performance profiles should only be 2Gi

Reply:
Although setting separate limits & requests will be a good idea, it will not go well with the pattern we have set here while giving resources for all the other daemons, where we have given the same requests & limits.

Instead of making so many complicated changes let me break down the change.
Previously mgr resource in 4.14- 1 CPU/3Gib memory
Now with this change
Lean- 0.5 CPU/2Gib memory
Balanced- 1 CPU/3 Gib memory(same as 4.14)
Performance- 2 CPU/4 Gib memory

Reply:
While we have had the pattern of all the components having the same requests and limits, I request that we change this pattern specifically for the mgr, to have the cpu/memory limits double the requests. (There are other components I want to change this pattern for as welll, but I will bring in up separately for a future release). As described in the original BZ comment (https://bugzilla.redhat.com/show_bug.cgi?id=2258357#c13), the mgr needs to burst, and also the standby mgr is almost not using any resources. Both the active mgr and standby are burstable in nature. As far as the QoS class, the Burstable class is fine for the mgr since there is not a concern to have it evicted to another node if necessary. Anyway, this change will not cause the QoS class of the mgr pod to change because it is already in the Burstable class.

Reply:
So basically 
```
requests<-->limits
Lean- 0.5 CPU/1 Gib memory<-->1 CPU/2 Gib memory
Balanced- 1 CPU/1.5 Gib memory(same as 4.14)<-->2 CPU/3 Gib memory
Performance- 1.5 CPU/2 Gib memory<-->3 CPU/4 Gib memory

```
@travisn please confirm the numbers

Reply:
/unhold